src/main/kotlin/ink/abb/pogo/scraper/tasks/Evolve.kt

```diff
@@ -0,0 +1,36 @@
+package ink.abb.pogo.scraper.tasks
+
+import POGOProtos.Enums.PokemonFamilyIdOuterClass
+import POGOProtos.Enums.PokemonIdOuterClass
+import com.pokegoapi.api.pokemon.Pokemon
+import ink.abb.pogo.scraper.Bot
+import ink.abb.pogo.scraper.Context
+import ink.abb.pogo.scraper.Settings
+import ink.abb.pogo.scraper.Task
+import ink.abb.pogo.scraper.evolve.EvolutionStrategy
+import ink.abb.pogo.scraper.evolve.IvMaximizingStrategy
+import ink.abb.pogo.scraper.util.Log
+
+/**
+ * Created by ddcbdevins on 7/26/16.
+ */
+class Evolve : Task {
+
+    private val DEFAULT_EVOLUTION_STRATEGY = "max_iv"
```

Comment:
The implementation in #939 should be the default implementation but I just put mine in here as a placeholder